build_tooling/storage_common.py

```diff
@@ -0,0 +1,51 @@
+from arcticdb import Arctic
+import pandas as pd
+import os
+import numpy as np
+
+LIBRARIES = [
+    # LINUX
+    "linux_cp36",
+    "linux_cp37",
+    "linux_cp38",
+    "linux_cp39",
+    "linux_cp310",
+    "linux_cp311",
+    # WINDOWS
+    "windows_cp37",
+    "windows_cp38",
+    "windows_cp39",
+    "windows_cp310",
+    "windows_cp311",
+]
+
+def real_s3_credentials():
```

Comment:
You don't need the individual components, so why not just have one environment variable that's the entire URI?

Reply:
We need the individual components for the versions_store, so I am just reusing the env variables